org.gnome.gThumb.yml

```diff
@@ -134,10 +134,10 @@ modules:
     buildsystem: meson
     sources:
       - type: archive
-        url: https://gitlab.gnome.org/GNOME/gthumb/-/archive/3.12.0/gthumb-3.12.0.tar.gz
-        sha256: 9b0de513a751de5f0fedf37e8d04ef631bf58fe05639f651d0208caccfdbc0b3
+        url: https://gitlab.gnome.org/GNOME/gthumb/-/archive/3.12.2/gthumb-3.12.2.tar.gz
```

Comment:
Please use https://download.gnome.org/sources/gthumb/3.12/gthumb-3.12.2.tar.xz

Reply:
that doesn't address the build failure. It seem to stil want appstreamcli to generate the appstream file.

Reply:
I mean sure, but it is still a good opportunity to fix that

Reply:
@hfiguiere can you change the URL to download.gnome.org? Then I think you can merge this PR.
Thanks

Reply:
done